docs/installation/kind.md

```diff
@@ -16,39 +16,72 @@ KIND (Kubernetes in Docker) deployment of OVN kubernetes is a fast and easy mean
 - [kubectl]( https://kubernetes.io/docs/tasks/tools/install-kubectl/ )
 - Python and pip
 - jq
+- openssl
+- openvswitch
 
 **NOTE :**  In certain operating systems such as CentOS 8.x, pip2 and pip3 binaries are installed instead of pip. In such situations create a softlink for "pip" that points to "pip2".
 
-### Run the KIND deployment with docker
-
 For OVN kubernetes KIND deployment, use the `kind.sh` script.
 
 First Download and build the OVN-Kubernetes repo: 
 
 ```
-$ go env -w GO111MODULE=auto
-$ go get github.com/ovn-org/ovn-kubernetes; cd $(go env GOPATH)/src/github.com/ovn-org/ovn-kubernetes
+git clone github.com/ovn-org/ovn-kubernetes; 
+cd ovn-kubernetes
 ```
-
 The `kind.sh` script builds OVN-Kubernetes into a container image. To verify
 local changes before building in KIND, run the following:
 
 ```
 $ pushd go-controller
 $ make
 $ popd
+```
+
+### Run the KIND deployment with docker
+
+Build the image for fedora and launch the KIND Deployment
+
+```
+$ pushd dist/images
+$ make fedora
+$ popd
+
+$ pushd contrib
+$ export KUBECONFIG=${HOME}/ovn.conf
+$ ./kind.sh
```

Comment:
just running kind was supposed to help with this, doing make fedora is not necessary per say, kind I think build's the image from local branch and loads that into docker

Reply:
Did some replication testing on this issue aswell. Not sure if its OS specific but for my computer it would fail to execute kind.sh and timeout if make fedora was not executed prior

Reply:
its probably OS specific but strange that I use fedora too and don't need to do that. either ways we can always change the docs if someone else feels this is not correct, specially considering anyways over time we need to keep updating the installation docs.